src/java/org/apache/cassandra/tools/nodetool/Sjk.java

```diff
@@ -234,15 +221,11 @@ public boolean run(final NodeProbe probe, final Output output)
                 {
                     printUsage(parser, out, parser.getParsedCommand());
                 }
-                return true;
             }
             catch (Throwable e)
             {
                 e.printStackTrace(err);
             }
-
```

Comment:
do we ignore failure propagation and potentially change a result code here?

Reply:
Fixed.